packages/plugin-phone/test/integration/spec/call.js

```diff
@@ -149,11 +149,45 @@ describe(`plugin-phone`, function() {
     });
 
     describe(`#to`, () => {
-      it(`represents the receiving party`);
+      let call;
+      afterEach(() => {
+        const c = call;
+        call = undefined;
```

Comment:
because we need to do two things in this function: unset call and return a promise. the alternative is

```javascript
const promise = c.hangup...
call = undefined;
return promise
```